Data-Structures/HashTable/HashTable.js

```diff
@@ -0,0 +1,105 @@
+export default class HashTable {
+  constructor(limit = 100) {
+    // Initialize the storage and limit variables
+    this.storage = new Array(limit)
+    this.limit = limit
+  }
+
+  // Hash function:
+  _hash(key, max) {
+    // Initialize the hash variable to 0
+    let hash = 0
+    // Iterate through the key and add the character code at each iteration to the hash variable
+    for (let i = 0; i < key.length; i++) hash += key.charCodeAt(i)
+    // Return the hash modulo the max
+    return hash % max
+  }
+
+  // Insert a key-value pair into the hash table
+  set(key, value) {
+    // Hash the key
+    const index = this._hash(key, this.limit)
+    // If the index is empty, insert the key-value pair and create a bucket
+    if (this.storage[index] === undefined) this.storage[index] = [[key, value]]
+    else {
+      //  If the index is not empty, iterate through the bucket (collision handling)
+      let inserted = false
+      for (let i = 0; i < this.storage[index].length; i++) {
+        if (this.storage[index][i][0] === key) {
+          // If the key exists, update the value
+          this.storage[index][i][1] = value
+          inserted = true
```

Comment:
Just early `return` here. Then you don't need the `inserted` variable, and you will be more efficient (by not doing any further loop iterations).